### DISCUSSION TITLE
Doc comment on Semantic search (/developers/academy/py/starter_text_data/text_searches/semantic)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1890] [Docs/blog comments]
# Doc comment on Semantic search (/developers/academy/py/starter_text_data/text_searches/semantic)

**Comment by shoff** [comment]
Would be nice to be shown how the collection was created, especially in python v4

**Reply by databyjp** [reply]
Hi @shoff - it's shown here :) https://weaviate.io/developers/academy/py/starter_text_data/text_collections/create_collection
